<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI::AddScope</class>
 <widget class="QDialog" name="GUI::AddScope">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>354</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create new scope</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblScopeName">
       <property name="text">
        <string>Scope &amp;name</string>
       </property>
       <property name="buddy">
        <cstring>leScopeName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leScopeName">
       <property name="placeholderText">
        <string>Enter new scope name...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCloseDialog">
       <property name="text">
        <string>C&amp;lose dialog</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCreateScope">
       <property name="text">
        <string>&amp;Create scope</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCloseDialog</sender>
   <signal>clicked()</signal>
   <receiver>GUI::AddScope</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCreateScope</sender>
   <signal>clicked()</signal>
   <receiver>GUI::AddScope</receiver>
   <slot>onCreateScope()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCreateScope</sender>
   <signal>clicked()</signal>
   <receiver>GUI::AddScope</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCreateScope()</slot>
 </slots>
</ui>
